docs/modules/localstack.md

```diff
@@ -62,6 +62,26 @@ Testcontainers will inform Localstack of the best hostname automatically, using
 
 * Other usage scenarios, such as where the Localstack container is used from both the test host and containers on a custom network are not automatically supported. If you have this use case, you should set `HOSTNAME_EXTERNAL` manually.
 
+## AWS Lambda and Networking
+
+When a test using Localstack triggers a [AWS Lambda function](https://aws.amazon.com/lambda/), Localstack [starts a new docker container](https://docs.localstack.cloud/localstack/lambda-executors/#docker) to run the Lambda code. This might cause problems if your Lambda function calls other AWS services in your Localstack rig. To fix this we have to make sure these Lambda containers are in the same network as the rest of our containers.
```

Comment:
```suggestion
When a test using Localstack triggers a [AWS Lambda function](https://aws.amazon.com/lambda/), Localstack [starts a new docker container](https://docs.localstack.cloud/localstack/lambda-executors/#docker) to run the Lambda code. This might cause problems if your Lambda function calls other AWS services in your Localstack setup. To fix this we have to make sure these Lambda containers are in the same network as the rest of our containers.
```

not sure if `setup` would be the right word but at least it is something I can understand 😅